content/docs/get-started/install/versions.md

```diff
@@ -18,6 +18,7 @@ The current stable version of Pulumi is **{{< latest-version >}}**.
         </tr>
     </thead>
     <tbody>
+        {{< changelog-table-row version="3.40.1" date="2022-09-19" showChecksum="true" >}}
```

Comment:
Note that the checksums for `3.40.0` (and `3.40.0-alpha.1662150830`) and now `3.40.1` are currently missing:

- https://get.pulumi.com/releases/sdk/pulumi-3.40.0-checksums.txt
- https://get.pulumi.com/releases/sdk/pulumi-3.40.1-checksums.txt

Tracked in https://github.com/pulumi/docs/issues/8043

If we wanted, we could temporarily set `showChecksum="false"` to hide those until we've addressed https://github.com/pulumi/docs/issues/8043

Reply:
Went ahead and added a commit that hides the checksum links for these versions temporarily. We can add the links back after we publish the checksum files.